### PR TITLE
[Refactor][doc] add data model and index doc

### PR DESCRIPTION
**Pull request by JNSimba**
## Problem Summary:
1. data model:Aggregate/Unique/Duplicate
2. index: bitmap/prefix/bloomfilter

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
3. Has unit tests been added: (Yes/No/No Need)
4. Has document been added or modified: (Yes/No/No Need)
5. Does it need to update dependencies: (Yes/No)
6. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: new-docs/zh-CN/data-table/data-model.md [hunk 1]
```diff
@@ -24,4 +24,430 @@ specific language governing permissions and limitations
...
+    `date` DATE NOT NULL COMMENT "数据灌入日期时间",
+    `city` VARCHAR(20) COMMENT "用户所在城市",
+    `age` SMALLINT COMMENT "用户年龄",
+    `sex` TINYINT COMMENT "用户性别",
+    `last_visit_date` DATETIME REPLACE DEFAULT "1970-01-01 00:00:00" COMMENT "用户最后一次访问时间",
+    `cost` BIGINT SUM DEFAULT "0" COMMENT "用户总消费",
+    `max_dwell_time` INT MAX DEFAULT "0" COMMENT "用户最大停留时间",
+    `min_dwell_time` INT MIN DEFAULT "99999" COMMENT "用户最小停留时间"
+)
+AGGREGATE KEY(`user_id`, `date`, `city`, `age`, `sex`)
```
Comment (hf200012) [inline new-docs/zh-CN/data-table/data-model.md:81]: give the full SQL

File: new-docs/zh-CN/data-table/data-model.md [hunk 2]
```diff
@@ -24,4 +24,430 @@ specific language governing permissions and limitations
...
+    `user_id` LARGEINT NOT NULL COMMENT "用户id",
+    `username` VARCHAR(50) NOT NULL COMMENT "用户昵称",
+    `city` VARCHAR(20) COMMENT "用户所在城市",
+    `age` SMALLINT COMMENT "用户年龄",
+    `sex` TINYINT COMMENT "用户性别",
+    `phone` LARGEINT COMMENT "用户电话",
+    `address` VARCHAR(500) COMMENT "用户地址",
+    `register_time` DATETIME COMMENT "用户注册时间"
+)
+UNIQUE KEY(`user_id`, `username`)
```
Comment (hf200012) [inline new-docs/zh-CN/data-table/data-model.md:267]: give the full SQL

File: new-docs/zh-CN/data-table/data-model.md [hunk 3]
```diff
@@ -24,4 +24,430 @@ specific language governing permissions and limitations
...
+CREATE TABLE IF NOT EXISTS example_db.expamle_tbl
+(
+    `timestamp` DATETIME NOT NULL COMMENT "日志时间",
+    `type` INT NOT NULL COMMENT "日志类型",
+    `error_code` INT COMMENT "错误码",
+    `error_msg` VARCHAR(1024) COMMENT "错误详细信息",
+    `op_id` BIGINT COMMENT "负责人id",
+    `op_time` DATETIME COMMENT "处理时间"
+)
+DUPLICATE KEY(`timestamp`, `type`)
```
Comment (hf200012) [inline new-docs/zh-CN/data-table/data-model.md:331]: give the full SQL

File: new-docs/zh-CN/data-table/data-model.md (hunk 2 above)
Comment (JNSimba) [reply new-docs/zh-CN/data-table/data-model.md:269]: already modified

File: new-docs/zh-CN/data-table/data-model.md [hunk 4]
```diff
@@ -24,4 +24,438 @@ specific language governing permissions and limitations
...
+DISTRIBUTED BY HASH(`type`) BUCKETS 1
+PROPERTIES (
+"replication_allocation" = "tag.location.default: 1"
+);
+```
+
+这种数据模型区别于 Aggregate 和 Unique 模型。数据完全按照导入文件中的数据进行存储，不会有任何聚合。即使两行数据完全相同，也都会保留。 而在建表语句中指定的 DUPLICATE KEY，只是用来指明底层数据按照那些列进行排序。（更贴切的名称应该为 “Sorted Column”，这里取名 “DUPLICATE KEY” 只是用以明确表示所用的数据模型。关于 “Sorted Column”的更多解释，可以参阅[前缀索引]()）。在 DUPLICATE KEY 的选择上，我们建议适当的选择前 2-4 列就可以。
```
Comment (hf200012) [inline new-docs/zh-CN/data-table/data-model.md:344]: 这种数据模型区别于 Aggregate 和 Unique 模型。数据完全按照导入文件中的数据进行存储，不会有任何聚合。即使两行数据完全相同，也都会保留。 而在建表语句中指定的 DUPLICATE KEY，只是用来指明底层数据按照那些列进行排序。（更贴切的名称应该为 “Sorted Column”，这里取名 “DUPLICATE KEY” 只是用以明确表示所用的数据模型。关于 “Sorted Column”的更多解释，可以参阅[前缀索引]()）。在 DUPLICATE KEY 的选择上，我们建议适当的选择前 2-4 列就可以。
```suggestion
这种数据模型区别于 Aggregate 和 Unique 模型。数据完全按照导入文件中的数据进行存储，不会有任何聚合。即使两行数据完全相同，也都会保留。 而在建表语句中指定的 DUPLICATE KEY，只是用来指明底层数据按照那些列进行排序。（更贴切的名称应该为 “Sorted Column”，这里取名 “DUPLICATE KEY” 只是用以明确表示所用的数据模型。关于 “Sorted Column”的更多解释，可以参阅[前缀索引](../../data-table/index/prefix-index.html)）。在 DUPLICATE KEY 的选择上，我们建议适当的选择前 2-4 列就可以。
```

File: new-docs/zh-CN/data-table/index/prefix-index.md [hunk 5]
```diff
@@ -23,4 +23,56 @@ KIND, either express or implied.  See the License for the
...
+
+在 Aggregate、Unique 和 Duplicate 三种数据模型中。底层的数据存储，是按照各自建表语句中，AGGREGATE KEY、UNIQUE KEY 和 DUPLICATE KEY 中指定的列进行排序存储的。
+
+而前缀索引，即在排序的基础上，实现的一种根据给定前缀列，快速查询数据的索引方式。
+
+## 示例
+
+我们将一行数据的前 **36 个字节** 作为这行数据的前缀索引。当遇到 VARCHAR 类型时，前缀索引会直接截断。我们举例说明：
+
+1. 以下表结构的前缀索引为 user_id(8 Bytes) + age(4 Bytes) + message(prefix 20 Bytes)。
```
Comment (hf200012) [inline new-docs/zh-CN/data-table/index/prefix-index.md:42]:
```suggestion
1. 以下表结构的前缀索引为 user_id(8 Bytes) + age(4 Bytes) + message(prefix 24 Bytes)。
```

**Comment by JNSimba** [comment]
thanks,already modified

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**